styles/styles.css

```diff
@@ -0,0 +1,278 @@
+/* stylelint-disable no-missing-end-of-source-newline */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+::marker {
+  font-size: 14px;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+  color: #828282;
+  font-size: 8pt;
+  font-family: Verdana, Geneva, sans-serif;
+}
+
+main {
+  width: 85%;
+  height: 100vh;
+  background: #f6f6ef;
+  display: flex;
+  flex-direction: column;
+  margin: 8px;
+}
+
+.page {
+  display: grid;
+  grid-auto-rows: max-content auto;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.page__header {
+  grid-column: 1/3;
+  display: grid;
+  justify-content: start;
+  grid-template-columns: auto auto;
+  align-items: center;
+}
+
+.page__news {
+  grid-column: 1;
+  grid-row: 2;
+}
+
+.page__form {
+  grid-column: 2;
+  grid-row: 2;
+}
+
+@media only screen and (min-width: 300px) and (max-width: 750px) {
```

Comment:
lets keep main rules in one place
move lines 63-67 right below line 37

Reply:
fixed